fs/read.js

```diff
@@ -5,18 +5,87 @@ var fs = require('fs');
 
 // 1. Write script to read file theory.md and console the output buffer.
 
+var printContent = fs.readFile("./theory.md",(err,content)=>{
```

Comment:
Since readFile is async , it will return a callback function and you don't have to store it in a variable.
We store something in variable when it returns immediately and we are going to use the variable.
You can directly do
```js
  fs.readFile('./theory.md', (err, content) => {})
```